app/src/main/kotlin/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -1127,7 +1124,7 @@ class WebViewActivity :
         }
         sendExternalBusMessage(
             ExternalBusMessage(
-                id = json.get("id"),
+                id = json["id"],
```

Comment:
I missed this ... But currently this crashes because it is a JsonElement which is not properly handle by our custom serializer. I'm going to fix it in a new PR.

Reply:
Nvm the issue is not with JsonElement not handle. It is handled. The issue is that in some case JsonObject contains JsonLiteral that are not serializables. (it doesn't trigger on this PR but it does in my PR https://github.com/home-assistant/android/pull/5622 where I use `get("id")` and not assuming "id" as a Int.